tests/unit/test_api_urls.py

```diff
@@ -137,12 +143,6 @@ def test_api_comments_link_header_auth(self):
             self.assertEqual(
                 rv.content_type, 'text/html')
 
-    def test_api_set_labels_without_auth(self):
```

Comment:
Moved this test to the bottom of the file to have all related tests in one place https://github.com/webcompat/webcompat.com/pull/3609/files#diff-688eb42ddc0072d66da1cbe00c95cdef58f51e6cf2b058186be3215c53eebf1dR240

Reply:
good idea.